docs/.vuepress/components/ActionBarDoc.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+	<StackLayout>
+		<ActionBar :title="title">
+			<ActionItem @tap="onTapShare($event)" ios.systemIcon="9" ios.position="left" android.systemIcon="ic_menu_share" android.position="actionBar" :webIcon="'fa ' + selectedShareIcon" />
+			<ActionItem @tap="onTapDelete($event)" ios.systemIcon="16" ios.position="right" text="delete" android.position="popup" :webIcon="'fa ' + selectedDeleteIcon" />
```

Comment:
I think we should remove ios, android attributes in doc file

Reply:
Removed iOS & Android attributes